tpot/base.py

```diff
@@ -630,11 +625,11 @@ def _update_top_pipeline(self):
 
             if not self._optimized_pipeline:
                 raise RuntimeError('There was an error in the TPOT optimization '
-                                  'process. This could be because the data was '
-                                  'not formatted properly, or because data for '
-                                  'a regression problem was provided to the '
-                                  'TPOTClassifier object. Please make sure you '
-                                  'passed the data to TPOT correctly.')
+                                   'process. This could be because the data was '
+                                   'not formatted properly, or because data for '
+                                   'a regression problem was provided to the '
+                                   'TPOTClassifier object. Please make sure you '
+                                   'passed the data to TPOT correctly.')
```

Comment:
Thank you for the various formatting changes. However, as a note for future PRs, beware of making numerous changes like these ones when they are unrelated to the core PR. They can cause merge conflicts if another PR happens to be changing those lines as well. I think we can make it work this time.

Reply:
Hmm, sorry for this. Think i might have accidently hit ‘auto format’ in my ide. Hopefully wont be harmfull in any way. Let me know if i should roll back Some things.

Reply:
No worries. I'm pretty sure we can sort out any merge conflicts this time because there's only one other PR being actively developed (just merged into the dev branch).